playbooks/5_setup_bastion.yaml

```
@@ -77,7 +78,8 @@
     - set_firewall
     - { role: dns, when: env.bastion.options.dns }
     - { role: check_dns, when: env.bastion.options.dns is defined and env.bastion.options.dns }
-    - { role: haproxy, when: env.bastion.options.loadbalancer.on_bastion }
+    - { role: haproxy, when: env.bastion.options.loadbalancer.on_bastion and control_node_count | int > 1}
```

Do not change this line.
This will break the existing SNO install using bootstrap node.
Your sno_haproxy role will overwrite the happroxy setup anyhow, if it is used.
Or you add this change:
```suggestion
    - { role: haproxy, when: env.bastion.options.loadbalancer.on_bastion and abi.flag is not defined and abi.flag }
```

---

Ok I will do the changes and test it and commit once it work as expected. Thank you

---

Removed check for haroxy as sno_haproxy role will overrite the outcomes. Please have a look at this dbdc74e3b8e8d0c80c52c5843e5a9720d2c2444c